Install Failure

Hi @danmarshall ,
I'm having issues installing tsx-create-element@2.2.4:

```
> tsx-create-element@2.2.4 preinstall D:\work\icg\workspace\lux-sanddance\node_modules\tsx-create-element
> npx npm-force-resolutions

npx: installed 5 in 1.448s
ENOENT: no such file or directory, open './package-lock.json'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\webpack-dev-server\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\watchpack-chokidar2\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! tsx-create-element@2.2.4 preinstall: `npx npm-force-resolutions`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the tsx-create-element@2.2.4 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Apparently this might be caused by the recently added line in package.json: `preinstall: "npx npm-force-resolutions"`

See https://github.com/oktadeveloper/schematics/issues/192#issuecomment-688720326

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "tsx-create-element",
-  "version": "2.2.4",
+  "version": "2.2.5",
   "description": "Use TypeScript TSX without React",
   "main": "dist/es6/index.js",
   "types": "dist/es6/index.d.ts",
@@ -8,7 +8,7 @@
     "dist/es6/*"
   ],
   "scripts": {
-    "preinstall": "npx npm-force-resolutions",
+    "dev-force-resolutions": "npx npm-force-resolutions",
     "bundle": "npm run test && parcel build ./tmp/index.js --no-minify --no-source-maps --out-dir ./docs",
     "build": "tsc -p .",
     "watch": "tsc -p . -w",

